pom.xml

```diff
@@ -816,6 +824,10 @@ under the License.</licenseText>
       </plugins>
     </pluginManagement>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-pmd-plugin</artifactId>
+      </plugin>
```

Comment:
```suggestion
```
why profile not working? its done like in quarkus:
https://github.com/quarkusio/quarkus/blob/d8618eccda1af323726537ba25bdbd24cdcee280/build-parent/pom.xml#L693

Reply:
See https://maven.apache.org/pom.html#Activation and https://issues.apache.org/jira/browse/MNG-4917

The activeByDefault flag might act a bit suprising sometimes, as it is only applied, if no other profile is selected explicitly (e.g. via -P flag or in settings.xml or any other profile that is e.g. activated by property automatically).

Reply:
yes, spot is `activeByDefault` from parent pom, so this might be the collision.

As root cause is bug, it seems justiciable to accept this temporary.